framework/python/src/common/risk_profile.py

```diff
@@ -33,27 +33,29 @@ def __init__(self, profile_json=None, profile_format=None):
     if profile_json is None or profile_format is None:
       return
     self.name = profile_json['name']
-    self.created = datetime.now()
+    self.created = datetime.now().strftime('%Y-%m-%d')
     self.version = profile_json['version']
```

Comment:
Keep this as a timestamp, no need to convert to string outside o to_json method.

Reply:
The issue with using a timestamp, is that when it is passed to the UI, it passes it as a full timestamp in ISO format but I was wanting to pass it to the UI as a string in a particular format.